google/api_core/timeout.py

```diff
@@ -15,7 +15,8 @@
 """Decorators for applying timeout arguments to functions.
 
 These decorators are used to wrap API methods to apply either a constant
-or exponential timeout argument.
+(DEPRECATED), exponential (DEPRECATED) or Deadline-dependent (recommended)
```

Comment:
put the recommended one first:
"to apply a Deadline-dependent (RECOMMENDED), constant (DEPRECATED), or exponential (DEPRECATED) "

Reply:
Flipped the order